apps/settings/src/router.js

```diff
@@ -53,10 +53,25 @@ export default new Router({
 			component: Users,
 			props: true,
 			name: 'users',
+			meta: {
+				title: () => {
+					return t('settings', 'Active users')
+				},
```

Comment:
Does it need to be a function?

```suggestion
				title: t('settings', 'Active users')
```